packages/authorization-process/src/routers/AuthorizationRouter.ts

```diff
@@ -47,7 +50,29 @@ const authorizationRouter = (
     .post(
       "/clientsConsumer",
       authorizationMiddleware([ADMIN_ROLE]),
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        const ctx = fromAppContext(req.ctx);
+        try {
+          const { client, showUsers } =
+            await authorizationService.createConsumerClient(
+              req.body,
+              ctx.authData.organizationId,
+              req.ctx.correlationId,
+              ctx.logger
+            );
+          return res
+            .status(200)
+            .json(clientToApiClient(client, { includeKeys: false, showUsers }))
+            .end();
+        } catch (error) {
+          const errorRes = makeApiProblem(
+            error,
+            createClientErrorMapper,
+            ctx.logger
+          );
+          return res.status(errorRes.status).json(errorRes).end();
```

Comment:
```suggestion
          return res.status(500).send();
```
You can just return 500 and delete the mapper since it doesn't have any other errors to return

Reply:
@AsterITA I don't think so. If remove the whole `makeApiProblem` we lose the logging. This should rather be:
```
const errorRes = makeApiProblem(error, () => 500, ctx.logger);
return res.status(errorRes.status).json(errorRes).end();
```